app/livechat/client/views/app/tabbar/visitorNavigation.js

```diff
@@ -30,10 +29,14 @@ Template.visitorNavigation.helpers({
 	},
 });
 
-Template.visitorNavigation.onCreated(function() {
+Template.visitorNavigation.onCreated(async function() {
 	const currentData = Template.currentData();
+	this.isLoading = new ReactiveVar(true);
+	this.pages = new ReactiveVar([]);
 
 	if (currentData && currentData.rid) {
-		this.pageVisited = this.subscribe('livechat:visitorPageVisited', { rid: currentData.rid });
+		const { pages } = await APIClient.v1.get(`livechat/visitors.pagesVisited?roomId=${ currentData.rid }`);
```

Comment:
We need to use pagination here, otherwise, only the first fetch will be displayed.